cassandra-diagnostics-ft/src/functional-test/resources/riemann/riemann.config

```diff
@@ -0,0 +1,46 @@
+; -*- mode: clojure; -*-
+; vim: filetype=clojure
+
+(logging/init {:console true})
+
+; Listen on the local interface over TCP (5555), UDP (5555), and websockets
+; (5556)
+(let [host "0.0.0.0"]
+  (tcp-server {:host host})
+  (udp-server {:host host})
+  (ws-server  {:host host}))
+
+; Expire old events from the index every 5 seconds.
+(periodically-expire 5)
+
+(def influx
+  (batch 100 1/10
+         (async-queue! :agg {:queue-size 1000
+                             :core-pool-size 1
+                             :max-pool-size 4
+                             :keep-alive-time 60000}
+
+            (influxdb {:host "influxdb"
+                       :version :0.9
+                       :port 8086
+                       :db "diagnostics-test"
+                       :username "admin"
+                       :password "secret"
+                       :tag-fields #{:host :id :statementType}}))))
+
+; Divide timestamp with 1000 if it is sent as ms to correctly handle ms precision
+(defn autoscale [ts]
+  (if (> ts 1000000000000) (/ ts 1000) ts))
+
+(let [index (index)]
+  (streams
+    (default :ttl 60
+
+    (adjust [:time autoscale]
+      index
```

Comment:
index, prn and expired streams are not required by this test. They can be removed.